public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -34,7 +34,7 @@ const PanelInspectorUnconnected = ({ panel, dashboard, plugin }: Props) => {
   });
 
   const location = useLocation();
-  const { data, isLoading, error } = usePanelLatestData(panel, dataOptions, true);
+  const { data, isLoading, error } = usePanelLatestData(panel, dataOptions, false);
```

Comment:
This fixes an issue where data tab can get stuck in loading state 
![image](https://github.com/grafana/grafana/assets/10999/43c14490-1fc4-498b-9ca7-dc45d8b98dde)